tests/tests_integration/test_api/test_iam.py

```diff
@@ -80,6 +82,9 @@ def test_create_and_delete(self, cognite_client):
 
 
 class TestSessionsAPI:
+    @pytest.mark.skipif(
+        os.environ.get("LOGIN_FLOW") == "client_certificate", reason="Sessions do not work with client_certificate"
```

Comment:
Tip: this has a shortcut:
```suggestion
        os.getenv("LOGIN_FLOW") == "client_certificate", reason="Sessions do not work with client_certificate"
```